### DISCUSSION TITLE
eByte-Module

### DISCUSSION THREAD

**Discussion by wunderbaum** [dewenni/ESP32-Jarolift-Controller#15] [General]
Heute sind hier eByte-Module angekommen die mit dem CC1101 kompatibel sein sollen, angeblich aber von verlässlicherer Qualität.
Ja, Huch! Sind die winzig: :[https://www.cdebyte.com/pdf-down.aspx?id=1932](http://www.cdebyte.com/pdf-down.aspx?spm=a2g0o.detail.1000023.2.2b34i1kXi1kX8Z&id=1932)

Ich meine, es steht schon bei Ali in der Beschreibung aber wenn man es vor sich liegen hat ist es schon echt mickrig.
1.27er Raster, da braucht man scharfe Augen. Wo bekomme ich die nun her :-)

![ebyte](https://github.com/user-attachments/assets/7318c637-2589-4cd2-b2ae-3beec30b2493)

Ich werde die kleine Pille die nächsten Tage mal versuchen zu verdrahten. Vielleicht kann man auf der Basis dann eine Platine entwerfen.

**Comment by wunderbaum** [comment]
Es ist endlich vollbracht und funktioniert!
![PXL_20250127_171306766](https://github.com/user-attachments/assets/6fba1023-77cb-4c4a-bb8a-4c81e97b83b4)

Die Antenne ist mit einem 17,3cm-Kabel improvisiert und reicht zumindest erstmal die 2 Meter vom Schreibtisch zum Rollladen neben mir. Der ESP ist ein  ([ESP32-Super-Mini](https://www.aliexpress.com/item/1005006599545782.html)).

Damit ist zumindest erwiesen das die eByte-Module auch einsetzbar sind. Ob die nun wirklich besser sind in der Praxis sei dahingestellt.

**Reply by dewenni** [reply]
Danke für den Test!
Der ESP32-SuperMini ist ja auch interessant. Hat der bei der Größe auch eine WLAN Antenne onboard?
Es gibt ja auch eine "Plus" Variante mit Antennenanschluss, aber bei der normalen Variante geht das ohne?

**Reply by wunderbaum** [reply]
Ja, der ESP hat eine kleine Keramik-Antenne auf dem Board.
Achtung! Es gibt welche mit schwarzer Antenne, die senden gerade mal gar nichts, die mit den roten Antennen sind besser. Allerdings auch sehr sensibel.
Ich habe davon einige in diversen Projekten im Einsatz (einige WLED, Stromzähler etc). Kosten grad mal 1,5-2€,
Die laufen (bei mir) wesentlich besser wenn man die Pin-Beinchen nach oben einlötet.
Nun habe ich noch einen [Ebyte mit externer Antenne](https://www.aliexpress.com/item/1005003722357358.html) liegen.

Das wäre was für ein designtes Board. Aber mit Kicad komm ich grad so gar nicht klar.

**Reply by wunderbaum** [reply]
> Plus"

Sehe ich jetzt erst den mit _Plus_ und externer Antenne. Der kostet ja richtig Geld ;-) Ich habe die Minis in diversen Sales für grad 1,70€ gekauft. Da sind die CC1101 teurerer. Die scheinen mir in diesem Projekt aber wichtiger zu sein, vor allem weil man so gar nicht weiß ob:

1. was gesendet wird
2. es richtig gesendet wird
3. es ankommt
4. usw.

Die _Gemeinde_ die solche Motoren einsetzt, sie über MQTT ansteuern möchte und Ansprüche hat ist wohl eher klein vermute ich.

**Comment by schnauzerjade** [comment]
Guten Tag
bin jetzt erst auf dieses Projekt gestossen. Aber, was mich ernsthaft wundert, wenn ich das Projekt 'clone' und versuche zu kompilieren, geht es zum verrecken nicht. Dutzende Fehler innerhalb der gelistteten Libraries in der Art, das Variablen nicht konsistent gebraucht werden. Was mache ich da falsch ? Ich benutze Visusal Studio Code Version 1.109.5 mit platformIO 3.3.4 (2025-01-11) und pioarduino
